sky/shell/platform/android/platform_view_android.cc

```diff
@@ -401,6 +401,19 @@ void PlatformViewAndroid::SurfaceCreated(JNIEnv* env,
   NotifyCreated();
 
   SetupResourceContextOnIOThread();
+
+  UpdateThreadPriorities();
+}
+
+void PlatformViewAndroid::UpdateThreadPriorities() {
+  blink::Threads::Gpu()->PostTask(
+      []() { ::setpriority(PRIO_PROCESS, gettid(), -3); });
+
+  blink::Threads::UI()->PostTask(
+      []() { ::setpriority(PRIO_PROCESS, gettid(), -2); });
+
+  blink::Threads::IO()->PostTask(
+      []() { ::setpriority(PRIO_PROCESS, gettid(), -1); });
```

Comment:
Do we need to set the priority of the IO thread?  It's supposed to be for bulk background work.